mcp/node/chain.cpp

```diff
@@ -1495,60 +1506,72 @@ void mcp::chain::send_approve(std::shared_ptr<ApproveQueue> aq_a)
 {
 	ApproveSkeleton as;
 	as.epoch = mcp::approve::calc_elect_epoch(m_last_summary_mci);
-    LOG(m_log.info) << "[send_approve] m_last_summary_mci=" << m_last_summary_mci << " epoch=" << as.epoch;
+	LOG(m_log.info) << "[send_approve] m_last_summary_mci=" << m_last_summary_mci << " epoch=" << as.epoch;
 
 	mcp::db::db_transaction transaction(m_store.create_transaction());
 	mcp::block_hash hash;
 	std::string msg;
-	if(as.epoch <= 2){
+	if (as.epoch <= 2)
+	{
 		msg = mcp::block_hash(0).hex();
 	}
-	else{
-		bool exists(!m_store.stable_block_get(transaction, (as.epoch-2)*epoch_period, hash));
+	else
+	{
+		bool exists(!m_store.stable_block_get(transaction, (as.epoch - 2) * epoch_period, hash));
 		assert_x(exists);
 		msg = hash.hex();
 	}
-	
-	//char msg[3]={0x31,0x32,0x33};
+
+	// char msg[3]={0x31,0x32,0x33};
 	as.proof.resize(81);
-	auto* ctx = mcp::encry::get_secp256k1_ctx();
+	auto *ctx = mcp::encry::get_secp256k1_ctx();
 	secp256k1_pubkey rawPubkey;
 	auto w_secret = m_witness->witness_secret();
 	if (!secp256k1_ec_pubkey_create(ctx, &rawPubkey, w_secret.data()))
-        return;
-	
-	std::array<byte, 65> serializedPubkey;
-    unsigned char output[32]={0};
-    auto serializedPubkeySize = serializedPubkey.size();
-    secp256k1_ec_pubkey_serialize(
-        ctx,
+		return;
+
+	std::array<CryptoPP::byte, 65> serializedPubkey;
```

Comment:
@metav this complaint with new language standard, c++17, we can't just use "byte" or either uint8_t